src/tests/suite/daos_rebuild_ec.c

```diff
@@ -1422,7 +1423,7 @@ rebuild_ec_combined_ops(void **state)
 	if (!test_runable(arg, 6))
 		return;
 
-	rc = daos_pool_set_prop(arg->pool.pool_uuid, "self_heal", "delay_rebuild");
+	rc = daos_pool_set_prop(arg->pool.pool_uuid, "self_heal", "exclude\\;delay_rebuild");
```

Comment:
are any other tests impacted? for example, in daos_rebuild.c there are two test functions rebuild_delay_and_reintegrate() and rebuild_delay_and_extend() that also set self_heal to "delay_rebuild".